frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -20,19 +20,20 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
     },
   );
 
-export const OCS_VERSION_4_4_FLAG = 'OCS_VERSION_4_4';
+export const OCS_VERSION_4_5_AND_ABOVE_FLAG = 'OCS_VERSION_4_5_AND_ABOVE_FLAG';
```

Comment:
If you look at the flag name the value does not have flag in it.

Reply:
We are detecting 4_5. No need for the above part.

Reply:
ack